.travis.yml

```diff
@@ -27,29 +24,13 @@ branches:
     - master
     - 2.5-stable
 
-matrix:
-  allow_failures:
-    - rvm: ruby-head
-  exclude:
-    - rvm: jruby-9.1.16.0
-      env: MONGODB=2.6.12
-    - rvm: jruby-9.1.16.0
-      env: MONGODB=3.0.12
-    - rvm: jruby-9.1.16.0
-      env: MONGODB=3.2.11
-    - rvm: jruby-9.1.16.0
-      env: MONGODB=3.4.1
```

Comment:
It seems that L34 - L41 this `exclude` part does not work.
https://github.com/mongodb/mongo-ruby-driver/blob/master/.travis.yml#L33-L41

Because actually used env is `MONGODB_VERSION`. Not `MONGODB`.